packages/patternfly-4/react-core/src/components/Modal/examples/Modal.md

```diff
@@ -62,6 +64,58 @@ class SimpleModal extends React.Component {
 }
 ```
 
+```js title=Basic-with-description
+import React from 'react';
+import { Modal, Button } from '@patternfly/react-core';
+
+class SimpleModal extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isModalOpen: false
+    };
+    this.handleModalToggle = () => {
+      this.setState(({ isModalOpen }) => ({
+        isModalOpen: !isModalOpen
+      }));
+    };
+  }
+
+  render() {
+    const { isModalOpen } = this.state;
+
+    return (
+      <React.Fragment>
+        <Button variant="primary" onClick={this.handleModalToggle}>
+          Show Modal
+        </Button>
+        <Modal
+          title="Modal Header"
+          isOpen={isModalOpen}
+          onClose={this.handleModalToggle}
+          description="Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua."
```

Comment:
Can you change the description to match core so it's clear which text is the description? The text from core is:
> A description is used when you want to provide more info about the modal than the title is able to describe. The content in the description is static and will not scroll with the rest of the modal body.